api/v1/symphony.go

```diff
@@ -31,14 +30,6 @@ type SymphonySpec struct {
 	// Synthesizer refs must be unique across variations.
 	// Removing a variation will cause the composition to be deleted!
 	Variations []Variation `json:"variations,omitempty"`
-
-	// Bindings are inherited by all compositions managed by this symphony.
```

Comment:
We still haven't gotten rid of these in AKS, just the shared SynthesisEnv, is it a blocker for this change?

Reply:
lol fixed 😎